pkg/services/dashboardsnapshots/database/database.go

```diff
@@ -1,21 +1,36 @@
-package sqlstore
+package database
 
 import (
 	"context"
 	"time"
 
 	"github.com/grafana/grafana/pkg/components/simplejson"
+	"github.com/grafana/grafana/pkg/infra/log"
 	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/dashboardsnapshots"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
 	"github.com/grafana/grafana/pkg/setting"
 )
 
+type DashboardSnapshotStore struct {
+	sqlStore *sqlstore.SQLStore
```

Comment:
If you can use the interfaces instead, that would be nice. Preferably the lightweight DB interface rather than the heavy Store interface, but just going for one of those two interfaces would be an improvement, since they make the refactoring work easier down the line 🙂

```suggestion
	db db.DB
```

or

```
sqlStore sqlstore.Store
```